src/arch/x86_64/pci.rs

```diff
@@ -0,0 +1,50 @@
+use pci_types::{ConfigRegionAccess, PciAddress};
+use x86::io::*;
```

Comment:
We should use the `x86_64` crate instead of the `x86` crate. The kernel code also changed to `x86_64`: https://github.com/hermit-os/kernel/blob/main/src/arch/x86_64/kernel/pci.rs

Then we can remove the additional dependency.